### ISSUE TITLE
False positive when using two-argument form of `super`

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#5685]
Pyright emits a false positive error in the following code:

```python
class A:
    def __new__(cls) -> "A":
        return super(type, cls).__new__(cls)
```

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.323, which I just published. It will also be included in a future release of pylance.
